tests/L0/run_transformer/test_pipeline_parallel_fwd_bwd_ucc_async.py

```diff
@@ -0,0 +1,32 @@
+import os
+import logging
+
+import torch
+from torch.testing._internal import common_utils
+from torch.testing._internal import common_distributed
+
+from apex.transformer.testing.distributed_test_base import UccDistributedTestBase
+
+from test_pipeline_parallel_fwd_bwd import PipelineParallelForwardBackwardTestBase
+
+logging.getLogger("torch").setLevel(logging.WARNING)
+logging.getLogger("apex").setLevel(logging.WARNING)
+
+class UccPipelineParallelForwardBackwardUnderLoadTest(UccDistributedTestBase, PipelineParallelForwardBackwardTestBase):
+
+    def __init__(self, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs)
+        self.GLOBAL_BATCH_SIZE = 1024
+        self.MICRO_BATCH_SIZE = 64
+        self.HIDDEN_SIZE = 32
+        self.NUM_EPOCHS = int(os.getenv("NUM_EPOCHS", "1"))
```

Comment:
[optional] can we avoid this dependence on an env var? e.g. adding a new argument of `num_epochs` to the impl.